compiler/parser.mly

```diff
@@ -77,18 +80,26 @@ stmt:
 
 expr:
   | literal                     { $1 }
-  | ID LPAREN args_opt RPAREN   { Call($1, $3)}
```

Comment:
Are you sure this should be moved? It was the second thing in most other asts, and I believe order matters